scripts/initialize_project.sh

```diff
@@ -19,9 +19,9 @@ docker compose up -d
 echo "Setting IPs for Minio..."
 ip=$(docker inspect tq-data | grep IPAddress | grep --extended-regexp --only-matching "([0-9]{1,3}\.){3}[0-9]{1,3}")
 echo "Using IP: $ip"
-sed -i "s/TQ_S3_MEDIA_HOST: .*/TQ_S3_MEDIA_HOST: $ip/" overrides.yml
-sed -i "s/TQ_S3_STATIC_HOST: .*/TQ_S3_STATIC_HOST: $ip/" overrides.yml
-sed -i "s/TQ_S3_FINANCE_HOST: .*/TQ_S3_FINANCE_HOST: $ip/" overrides.yml
+sed -i '' "s/TQ_S3_MEDIA_HOST: .*/TQ_S3_MEDIA_HOST: $ip/" overrides.yml
```

Comment:
Thanks for the suggestion. I'm not sure if this also works on Linux @thgoebel can you maybe verify it works on both platforms? Thanks!